specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -749,6 +749,44 @@
         "x-ms-odata": "#/definitions/DeploymentExtendedFilter"
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/{resourceProviderNamespace}/register": {
+      "post": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "Providers_Register",
+        "description": "Registers a management group with a resource provider.",
+        "parameters": [
+          {
+            "name": "resourceProviderNamespace",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace of the resource provider to register."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
```

Comment:
there's a linter error on this parameter. If you look at the "GroupIdParameter" definition, it has name as "groupId", while your API signature has "managementGroupId".